src/js/core.js

```diff
@@ -899,7 +899,7 @@
 
             try {
                 this.events.disableCustomEvent('editableInput');
-                if (opts.url && opts.url.trim().length > 0) {
+                if (opts.url && opts.url.length > 0) {
```

Comment:
No.
See the change I made in src/js/extensions/anchor.js file. I am trimming down the url before setting in opts.url variable. So opts.url.length would be equal to 0 if it is only whitespace there.